commands/remind.js

```diff
@@ -17,23 +17,38 @@ function Remind(client, msg, args)  {
 
     const atIndex = args.lastIndexOf('at');
     const inIndex = args.lastIndexOf('in');
+    const mentionTest = /<@!?([0-9]+)>/;
+
+console.log(msg);
+console.log(client);
+console.log(args);
+console.log(mentionTest);
+mentionTest
```

Comment:
Prefer not to have `console.log`s in the code except where necessary (errors, stats, etc.).  Also, just calling `mentionTest` shouldn't actually do anything.